docs/reference/pytest-operator.md

```diff
@@ -0,0 +1,19 @@
+(pytest-operator)=
+# `pytest-operator`
```

Comment:
I wonder if we really need to have this page. There should certainly be side links to the pytest-operator and python-libjuju (python-libjuju is really the more important part) whenever the ops docs talk about testing, but it's a very separate project, so it doesn't seem to make sense here. We don't have a charmcraft or juju page, for example.

Reply:
:+1: tracked this in my actions doc